testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/auditlog/AuditLogBootingSyslogTest.java

```diff
@@ -60,6 +62,12 @@
     private static final AuditLogToTLSElytronSyslogSetup SYSLOG_SETUP = new AuditLogToTLSElytronSyslogSetup();
 
 
+    @BeforeClass
+    public static void noJDK12() {
+        Assume.assumeFalse("Avoiding JDK 12 due to https://bugs.openjdk.java.net/browse/JDK-8219658", "12".equals(System.getProperty("java.version")));
```

Comment:
Hi @bstansberry :)
Given that this property can have values like `12-ea`, `12.0.1` and such[1], maybe use `java.specification.version` instead for the String equality check?

[1] https://openjdk.java.net/jeps/223

P.S: (Belated) congratulations on becoming the WildFly project lead :)

Reply:
@jaikiran Hello, and thanks! Both for the suggestion and the congrats!